crates/interfaces/src/network.rs

```diff
@@ -0,0 +1,4 @@
+use reth_network_api::NetworkError;
+
+/// Error alias for Network error type
+pub type Error = NetworkError;
```

Comment:
this file is no longer needed and can be removed.

Reply:
Alright